Configuration/Geometry/test/BuildFile.xml

```diff
@@ -1,4 +1,4 @@
 <environment>
-  <test name="test2026Geometry" command="python3 ${LOCALTOP}/src/Configuration/Geometry/scripts/generate2026Geometry.py -t"/>
+  <test name="testRun4Geometry" command="python3 ${LOCALTOP}/src/Configuration/Geometry/scripts/generateRun4Geometry.py -t"/>
```

Comment:
suppose you need to rename this:

https://github.com/cms-sw/cmssw/blob/7d3d80e8707194c87090fd2ef9ae0319ea2cefa1/Configuration/Geometry/scripts/generate2026Geometry.py#L1-L7

too